### PR TITLE
HDDS-4971. Define Ozone specific trash-interval and checkpoint interval

### PR DESCRIPTION
**Pull request by sadanand48**
## What changes were proposed in this pull request?
This change adds 2 new configs in ozone. One thing to note here is that ozone trash is not fully independent from the hadoop-configurations i.e (`fs.trash.interval`) . This config (`fs.trash.interval`) will still control enabling/disabling of trash feature whereas the newly added configs (`ozone.fs.trash.interval` & `ozone.fs.trash.checkpoint.interval` ) would set values if trash is enabled . These will add flexibility if ozone user wants to set ozone specific interval period. 
 
## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-4971

## How was this patch tested?
Unit test

### CODE REVIEW

File: hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFileSystem.java [hunk 1]
```diff
@@ -798,10 +799,11 @@ public void testTrash() throws Exception {
     String testKeyName = "testKey2";
     Path path = new Path(OZONE_URI_DELIMITER, testKeyName);
     ContractTestUtils.touch(fs, path);
-    Assert.assertTrue(trash.getConf().getClass(
-        "fs.trash.classname", TrashPolicy.class).
+    Assert.assertTrue(trash.getConf().getClass("fs.trash.classname",
```
Comment (mukul1987) [inline hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFileSystem.java:802]: this line is an unrelated indentation related change ?

Comment (sadanand48) [reply hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFileSystem.java]: Yes, addressed comment.Thanks

**Review by mukul1987 — APPROVED** [review]
+1, the change looks good to me.
